<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Recherche_par_nom</class>
 <widget class="QMainWindow" name="Recherche_par_nom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit {
background: transparent;
	border : none;
	color:rgb(22, 204, 137);
	border-bottom: 1px solid  rgb(0, 120, 180);
font: 11pt &quot;Century Gothic&quot;;
}
QLabel{
	color: rgb(78, 156, 234);
font: 12pt &quot;Century Gothic&quot;;
}
QCheckBox{
	color: rgb(0, 170, 255);
	font: 11pt &quot;Arial Rounded MT Bold&quot;;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>801</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(230, 230, 230);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QCheckBox" name="checkBox_5">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>100</y>
       <width>76</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Age</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>160</y>
       <width>76</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>CIN</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>155</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>210</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 13pt &quot;Century Gothic&quot;;
color:white;
background-color: rgb(0, 170, 255);
;
border-radius:10px;
	</string>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>40</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 170, 255);
	font: 12pt &quot;Arial Rounded MT Bold&quot;;</string>
     </property>
     <property name="text">
      <string>Recherche des employers</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>135</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>80</y>
       <width>76</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>512</x>
       <y>90</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>120</y>
       <width>76</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Prenom</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>115</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_4">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>140</y>
       <width>76</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Salaire</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>75</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
